groups/alteryx-live/keys.tf

```diff
@@ -3,5 +3,5 @@
 # ------------------------------------------------------------------------------
 resource "aws_key_pair" "alteryx_keypair" {
   key_name   = "${var.application}-${var.application_environment}"
-  public_key = local.alteryx_ec2_data["public-key"]
-}
\ No newline at end of file
+  public_key = local.alteryx_ec2_data 
```

Comment:
It feels like the local var could be better named to represent what it's storing?